resources/faq.md

```diff
@@ -0,0 +1,163 @@
+# FAQ
+
+This FAQ is still a work in progress.<br>
+Do not hesitate to ask us your own questions. It will help us improve this page!
+
+## I have never used a search engine before. Can I use MeiliSearch anyway?
+
+Of course! No need to get used with ElasticSearch or Solr to handle MeiliSearch.
+
+MeiliSearch is really **easy to use** and is accessible for all kinds of developers!
+
+[A quick tour](/guides/introduction/quick_start_guide.md) is available in this documentation to get the basics of MeiliSearch.
+
+We also provide a lot of tools, like [our SDKs](/resources/sdks.md), to help you easily integrate MeiliSearch in your project. New tools arrive every day!
+
+Plus, you can [contact us](/resources/faq.md#how-can-i-contact-the-meilisearch-team) if you need any help. We will answer for sure!
+
+## Do I need to configure MeiliSearch to get it working?
+
+MeiliSearch configuration works out-of-the-box. It means, by default, MeiliSearch handles all the configurations needed to provide a powerful and relevant search.
+
+For example, without any configuration, MeiliSearch is typo tolerant.<br>
+Type `craete an inedx` in the search bar of this documentation to experience the typo tolerance of our search engine.
+
+If you want to understand the relevancy of MeiliSearch, here is detailed and well-explained [information about the subject](/guides/main_concepts/relevancy.md#ranking-rules).
+
+Of course, MeiliSearch is still [highly customizable](/references/settings.md) to adapt the search to your needs with synonyms, stop words, and custom ranking rules.
+
+## How to know if MeiliSearch perfectly fits my use cases?
+
+MeiliSearch is open-source and really easy to use. It means that you can easily do a quick test with your data following our [getting started](/guides/introduction/quick_start_guide.md).
```

Comment:
```suggestion
Since MeiliSearch is an open-source and easy-to-use tool, you can give it a try using your data. Follow this [guide](/guides/introduction/quick_start_guide.md) to get a quick start!
```